.github/CODEOWNERS

```diff
@@ -31,19 +31,14 @@
 
 # PRLabel: %Azure.Core
 # ServiceLabel: %Azure.Core
-# AzureSdkOwners: @xirzec @jeremymeng @deyaaeldeen @timovv @mpodwysocki
-/sdk/core/ @xirzec @jeremymeng @deyaaeldeen @timovv @mpodwysocki
+# AzureSdkOwners: @Azure/azure-sdk-for-js-core
+/sdk/core/ @Azure/azure-sdk-for-js-core
```

Comment:
You cannot use a team unless this team is a child of azure-sdk-write team. I took a quick look and that doesn't look to be the case.

Reply:
We gave the team write access, but do you want to reparent it under azure-sdk-write for us? I don't think I have permission to do that.

Reply:
I see I didn't know that. We will need to create new teams then.